client/src/components/MyMap.js

```diff
@@ -27,6 +49,7 @@ function MyMap() {
 						acc.push({
 							rhythm_code: rhythm.rhythm_code,
 							map_id: rhythm.map_id,
+							isOn: false,
```

Comment:
I'm glad you've added this here, it will make sure we don't accidentally write other values to this key in the future.

Reply:
I've changed the name to isDropdownShown for the key here as well